stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -114,6 +139,51 @@ public class ChannelListView : FrameLayout {
         configureDefaultMoreOptionsListener(context)
     }
 
+    /**
+     * Returns the inner [RecyclerView] that is used to display a list of channel list items.
+     *
+     * @return The inner [RecyclerView] with channels.
+     */
+    public fun getRecyclerView(): RecyclerView {
```

Comment:
The naming is taken from `MessageListView` as we already expose the inner RecyclerView there with  `MessageListView.getRecyclerView()`